package.json

```diff
@@ -17,7 +17,7 @@
     "eslint": "eslint . --ext html",
     "polylint": "polymer lint",
     "lint": "npm run eslint && npm run polylint",
-    "test": "polymer test --simpleOutput --expanded -l chrome",
+    "test": "polymer test  -l chrome",
```

Comment:
What is the reason for this change? If you think we need it, I guess it should be done on a separate commit. (And maybe under a different ticket, just to make things organized)

Reply:
not really, I was using this to debug, will revert thanks for the heads-up